package.json

```diff
@@ -8,8 +8,8 @@
     "start": "cross-env NODE_ENV=development webpack-dev-server --hot",
     "build": "cross-env NODE_ENV=production webpack --config webpack.config.prod.js",
     "// TESTS": "",
-    "test": "jest --notify",
-    "test:coverage": "jest --notify --coverage",
+    "test": "jest --notify --detectOpenHandles",
+    "test:coverage": "jest --notify --detectOpenHandles --coverage",
```

Comment:
--detectOpenHandles flag has some performance penalty apparently, but I guess it's easier than identifying the tests that aren't exiting cleanly

Reply:
I prefer it not complaining when the tests finish so happy to leave it as we do elsewhere